worlds/hk/Options.py

```diff
@@ -406,6 +406,10 @@ class ExtraPlatforms(DefaultOnToggle):
     """Places additional platforms to make traveling throughout Hallownest more convenient."""
 
 
+class SharedItems(Toggle):
+    """Places all Vanilla items as AP Items so they can be shared within one slot. This will inflate hint prices."""
```

Comment:
Not necessarily a fan of the name "SharedItems" as it's not clear what that means. I feel a name like SyncUnshuffledLocations or AddUnshuffledLocations or similar would be more ideal.

Also some verbiage tweaks for doccomment to make it more clear.
```suggestion
    """Adds non-randomized locations to the location pool, which allows syncing
    of location state with co-op or automatic collection via collect.
    
    Note: This will increase the number of location checks required to purchase
    hints to the total maximum.
    """
```